src/components/indicators/z-stepper-item/index.tsx

```diff
@@ -24,33 +24,40 @@ export class ZStepperItem {
   @Prop({attribute: "pressed"})
   pressed: boolean;
 
+  /**
+   * The checked state of the item.
+   */
+  @Prop({attribute: "checked"})
+  checked: boolean;
+
   /**
    * The disabled state of the item.
    */
   @Prop({attribute: "disabled"})
   disabled: boolean;
 
-  render(): HTMLAnchorElement | HTMLButtonElement {
-    if (this.href) {
-      return (
-        <a
-          href={!this.disabled && this.href}
-          class="stepper-item"
-        >
-          <div class="indicator">{this.index}</div>
-          <span>
-            <slot />
-          </span>
-        </a>
-      );
-    }
+  private getAttributes(): Record<string, unknown> {
+    const href = this.href && !this.pressed && !this.disabled ? {onClick: () => (location.href = this.href)} : {};
+    const role = this.href && !this.disabled ? {role: "link"} : {};
```

Comment:
Visto che questo dipende direttamente da `href` appena definito sopra, non sarebbe meglio fare 
```const role = href ? {role: "link"} : undefined;```?

Reply:
Anche perché direi che deve dipendere anche da pressed.. Quindi in generale 
```if (this.pressed) return {}```
all'inizio del metodo forse ci potrebbe stare?